### DISCUSSION TITLE
DRX & C-DRX

### DISCUSSION THREAD

**Discussion by laurent512** [srsran/srsRAN_4G#1225] [General]
Hello, 

Thank you for the great project you made. 
I successfully setup the 4G project from source and provide data (over 4G) to my COTS device. 

Is there a way to configure the DRX parameters ? 
https://www.sharetechnote.com/html/Handbook_LTE_DRX.html

Thank you in advance :)

**Comment by ismagom** [comment]
Hi, 

DRX is not supported unfortunately.
